<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EyeDiagramDialog</class>
 <widget class="QDialog" name="EyeDiagramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eye Diagram</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Trace Selection</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Transmission line trace:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="traceSelector"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Input Datastream Configuration</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Data rate:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SIUnitEdit" name="datarate"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Rise time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="SIUnitEdit" name="risetime"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Fall time:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="SIUnitEdit" name="falltime"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>High level:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="SIUnitEdit" name="highLevel"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Low level:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="SIUnitEdit" name="lowLevel"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Noise (RMS):</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="SIUnitEdit" name="noise"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Jitter (RMS):</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="SIUnitEdit" name="jitter"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Pattern length:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="patternLength">
          <property name="currentIndex">
           <number>7</number>
          </property>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>31</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>63</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>127</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>255</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>511</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1023</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2047</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Eye Calculation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Displayed cycles:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="displayedCycles">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Update when trace changes:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="updateOnTraceChange">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="update">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progress">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="EyeDiagramPlot" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
  <customwidget>
   <class>EyeDiagramPlot</class>
   <extends>QWidget</extends>
   <header>Tools/eyediagramdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
